cmd/kubeless/function/call.go

```diff
@@ -19,18 +19,28 @@ package function
 import (
 	"bytes"
 	"fmt"
+	"math/rand"
 	"strconv"
 	"strings"
 	"time"
 
-	"github.com/kubeless/kubeless/cmd/kubeless/version"
 	"github.com/kubeless/kubeless/pkg/utils"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/rest"
 )
 
+const letterBytes = "abcdefghijklmnopqrstuvwxyz0123456789"
+
+func getRandString(n int) string {
+	b := make([]byte, n)
+	for i := range b {
+		b[i] = letterBytes[rand.Intn(len(letterBytes))]
```

Comment:
Is it important that event-ids are only from this quite-restricted base36 character set?
(I expected to see eg standard or url-safe base64 here)

Reply:
not really, is there an easy (short) way of doing that? There may be a way I am unaware of.

Reply:
```go
import (
    "encoding/base64"
    "crypto/rand"
)

func getRandString(n int) (string, err) {
    b := make([]byte, n)
    if _, err := rand.Read(b); err != nil {
        return err
    }
    return base64.RawURLEncoding.EncodeToString(b)
}
```